### DISCUSSION TITLE
动态的表情分辨率问题

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#4244] [综合讨论 / General]
单纯提一嘴看下有什么解决方案，正确来说这个问题应该由B站来修复的，但是已经拖了好久不干事了

![image](https://github.com/the1812/Bilibili-Evolved/assets/14146754/8cf417c3-5af5-49e1-b5aa-10f320537020)
动态的表情居然只有40x40
![image](https://github.com/the1812/Bilibili-Evolved/assets/14146754/9d443641-e02a-4388-b353-e49cd25f014a)

评论区倒是100x100看起来没那么难受
![image](https://github.com/the1812/Bilibili-Evolved/assets/14146754/330f5480-535d-4aa2-8d6f-8832553d2008)
![image](https://github.com/the1812/Bilibili-Evolved/assets/14146754/9bbf09bd-837f-4418-b9f4-dce5fb1d0580)

**Comment by u2shana** [comment]
请问有没有计划的打算，B站好像真不打算改

**Reply by the1812** [reply]
还没有, 这个可能得把高分辨率图片的功能拆一部分出来才能用在动态这里

当然现在能搞的一个歪招就是装上高分辨率图片, 如果系统缩放太低 (`window.devicePixelRatio` 小于 1.25) 的话改一下组件的 scale 选项 (乘上后能达到 50px 就行), 然后弄个 Stylus 样式把尺寸调大
```js
bilibiliEvolved.settingsApis.getComponentSettings('imageResolution').options.scale = 1.25
```


![Snipaste_2023-09-23_17-16-06](https://github.com/the1812/Bilibili-Evolved/assets/26504152/fd88e77d-466d-40e5-a41e-144223c6b418)
